docker/generic/start_proxy.py

```diff
@@ -548,6 +548,13 @@ def make_argparser():
         Otherwise use ignored_query_parameters. Defaults to false.
         ''')
 
+    parser.add_argument(
+        '--connection_buffer_limit_bytes', action=None,
+        help='''
+        Configure the maximum amount of data that is buffered for each 
+        request/response body, in bytes. If not set, default is decided by
+        Envoy (currently 1 MiB). 
```

Comment:
Maybe no need to mention the number, otherwise:
- we need to keep it sync with envoy
- possibly mislead our users if we forget

Reply:
Agreed with Jason, not to mention the actual default value. maybe add a link to envoy doc for this setting.

Reply:
Wondering if we should prefix the flag name with --envoy-.  since this is specific envoy setting.  just like envoy-xff
and create a envoy specific flag section, and move it there.

Reply:
Done. I changed the start_proxy flag to prefix with Envoy. The config manager flag does not have that prefix for now.